examples/seismic/acoustic/gradient_example.py

```diff
@@ -1,169 +0,0 @@
-import numpy as np
```

Comment:
I'm always happy to purge the codebase, you know, but perhaps it's better to inform whoever has maintained this

anyway, as I said, I'm glad to drop redundant code

Reply:
All the example was just duplicates of `wavesolver` and `acoustic_example` that was really a nightmare to maintain and delayed quite a few PRs in the passed.

Reply:
I understand, that's why I basically said "I agree with what you're doing". Nevertheless, it is still important to inform the author of this code about what's gonna happen.